starter-code/app.js

```diff
@@ -8,6 +8,13 @@ const hbs          = require('hbs');
 const mongoose     = require('mongoose');
 const logger       = require('morgan');
 const path         = require('path');
+const session      = require('express-session');
+const Mongostore   = require('connect-mongo')(session)
+const flash        = require('connect-flash');
+const passport     = require('passport');
+const LocalStrategy = require('passport-local').Strategy
```

Comment:
This line isn't necessary because the strategy has been defined already in your passport.js file, you should remove it